Dockerfile

```diff
@@ -7,7 +7,10 @@ RUN apk add --no-cache bash \
         bzip2-dev \
         gcc \
         libc-dev \
-    && pip install --no-cache-dir gevent==1.1.2 flask==0.11.1 kafka_python==1.3.1 requests==2.10.0 cloudant==2.1.0
+        py-pip \
+        python-dev \
+        linux-headers \
+    && pip install --no-cache-dir gevent==1.1.2 flask==0.11.1 kafka_python==1.3.1 requests==2.10.0 cloudant==2.1.0 psutil
```

Comment:
We should probably stick `psutil` to a specific version so that we can have consistent deployments.